setup.py

```diff
@@ -28,7 +28,7 @@ def grep(attrname):
 
 install_requires = ['python-dateutil']
 if sys.version_info[0] < 3:
-    install_requires.append('backports.functools_lru_cache==1.2.1')
+    install_requires.append('backports.functools_lru_cache==1.4')
```

Comment:
Would it be possible to have here `backports.functools_lru_cache>=1.2.1`?

Reply:
I personally don't have any issue with >= (in fact, I think less stringent bounding in general is a good thing, until a conflict is found), however all the other documented requirements are a strict ==.

I will make the change and see what the maintainer says.